va/va.h

```diff
@@ -2181,6 +2181,24 @@ VAStatus vaCreateBuffer (
 );
 
 /**
+ * Create a buffer for given width & height
+ * get unit_size, pitch, buf_id for 2D buffer
+ * for permb qp buffer, it will return unit_size for one MB or LCU
+ * and the pitch for alignments 
+ * can call vaMapBuffer with this Buffer ID to get virtual address. 
+ */
+VAStatus vaCreateBuffer2(
+    VADisplay dpy,
+    VAContextID context,
+    VABufferType type,
+    unsigned int width,
+    unsigned int height,
+    unsigned int *unit_size,
+    unsigned int *pitch,
```

Comment:
I think this needs a clearer description of what it does.  Is it that you supply width and height, and based on those and the buffer type the values pointed to by unit_size and pitch are written with something which you need to know when mapping the buffer?  (An example might help.)

Reply:
yes, when you create 2D buffer, you certainly need the pitch for the alignment information, unit_size is another one, for example, we need 4 bytes to represent something of one MB, the unit_size should be 4

Reply:
Agree with @fhvwy, could you update the comment?

Reply:
accept, will add a example